commands/command_migrate.go

```diff
@@ -306,6 +308,6 @@ func init() {
 		cmd.PersistentFlags().BoolVar(&migrateEverything, "everything", false, "Migrate all local references")
 		cmd.PersistentFlags().BoolVar(&migrateSkipFetch, "skip-fetch", false, "Assume up-to-date remote references.")
 
-		cmd.AddCommand(importCmd, info)
+		cmd.AddCommand(importCmd, info, exportCmd)
```

Comment:
Nit; let's put `exportCmd` before the other two arguments, to keep this list alphabetized.